queries/nim/highlights.scm

```diff
@@ -427,8 +385,8 @@
 
 ((identifier) @type
   (#has-ancestor? @type type_expression)
-  (#not-has-ancestor? @type pragma_list)
-  (#set! "priority" 98)) ; for parameters in proc_type
+  (#not-has-ancestor? @type pragma_list))
+; NOTE: needs to be after @variable
```

Comment:
This will be costly, much more than most queries in this file

Reply:
you're talking about the has-ancestor?

Reply:
Yep
You basically are traversing the entire depth of the tree with (not-has-ancestor), so this may lead to slowdowns

Reply:
I knew that. I was just being a little dumb, I benchmarked this and it looked fine, but my benchmarking was done with the ts cli, which doesn't know the has-ancestor directive.